.travis.yml

```diff
@@ -1,5 +1,9 @@
 sudo: required
 
+language: node_js
+node_js:
+  - "8"
```

Comment:
What about backward compatibility to node 6 and 7? Without babel compiler other apps running below node v8 are not working, right?

Reply:
Yes although native Promise support has existed since node 4 - http://node.green/#ES2015-built-ins-Promise

Reply:
I test with ```FROM node:6.11-alpine``` locally and the tests passed.

Reply:
Me too 😉